MCRDP4SAMD-1050-SWI-Rules (#29)
mia-care/documentation: docs/p4samd/handbook/software_items.md

| 15 | [EU AI Act / FDA / GMLP] Risk Control Measures Documentation | Missing risk control rationale | AI-based Software Items must include a documented rationale and control measures for risk mitigation. |
| 16 | - | Missing repository ID | ID of the repository associated with the Software Item is needed to correctly detect software item implementation. |
| 17 |[IEC 62304 **4.3**] | Undocumented SWI Reclassification | Please justify the re-classification of the SWI with a rationale in the dedicated classification reason field, describing how the new SOFTWARE ITEMS are segregated so that they may be classified separately. |

> | 18 |[IEC 62304 **4.3**] | Possible SoftwareItem Misclassification | This software item has the lower safety class than at least one of its children. |

Reviewer: ```suggestion
| 18 |[IEC 62304 **4.3**] | Possible Software Item Misclassification | This software item has a safety class lower than at least one of its children. |
```
